server/data/incidentTypeConfiguration/validation.ts

```diff
@@ -50,12 +50,15 @@ function checkUnreachableQuestions<Q>(configGraph: Graph<Q>, dfsResult: DfsResul
   }
 }
 
-function buildConfigGraph(config: IncidentTypeConfiguration): Graph<string> {
+function buildConfigGraph(config: IncidentTypeConfiguration, errors: Error[]): Graph<string> {
```

Comment:
I'm not so sure populating errors is a concern of the function that build the graph...it doesn't feel right (if you know what I mean).

I appreciate we already loop the questions and have an `activeAnswers` variable in here...but it still doesn't feel right?

I'd almost prefer for this check to be done separately even if it means looping through the config's questions/answers another time...but I don't think performance is so much of a problem in this case, having separate validation checks for related things is possibly more important?

Don't know what you think. If you have strong opinions on this I'm happy to bow and make this go though tho.